src/ucp/core/ucp_ep.c

```diff
@@ -1050,11 +1050,8 @@ static void ucp_ep_config_init_attrs(ucp_worker_t *worker, ucp_rsc_index_t rsc_i
 
     iface_attr = ucp_worker_iface_get_attr(worker, rsc_index);
 
-    if ((iface_attr->cap.flags & short_flag) &&
-        (context->config.ext.enable_memtype_cache)) {
-        config->max_short = max_short - hdr_len;
-    } else {
-        config->max_short = -1;
+    if ((iface_attr->cap.flags & short_flag)) {
+	config->max_short = max_short - hdr_len;
```

Comment:
1. indentation
2. what would prevent ucp_tag_send_inline() from trying to send Cuda memory?

Reply:
if cache is "on",   ucp_memory_type_cache_is_empty() will prevent ucp_tag_send_inline()
if cache off,  ucp_tag_send_inline() is prevented for both CUDA and HOST in fastpath.  The inline will be tried for HOST only after memory type is detected in ucp_tag_send_req()